Makefile

```diff
@@ -64,4 +63,9 @@ publish-release: test
 		# $(INFO)Publishing release(END)
 		./gradlew :eppo:publishReleasePublicationToMavenRepository
 
+.PHONY: publish-release
+publish-release: test check-maven-credentials-and-publish
+
+.PHONY: publish-release-without-test
+publish-release-without-test:check-maven-credentials-and-publish
```

Comment:
Since, the test workflow is not working currently. I have bypassed that step for now.